.codeclimate.yml

```diff
@@ -0,0 +1,35 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  markdownlint:
+    enabled: true
+  eslint:
+    enabled: true
+    config:
+      config: .eslintrc.yml
+
+      extensions:
+      - .es6
+      - .js
+      - .jsx
+
+  fixme:
+    enabled: true
+    config:
+      strings:
+      - FIXME
+      - TODO
+      - XXX
+      - HACK
+ratings:
+  paths:
+  - "**.js"
+  - "**.md"
+  - "**.es6"
+  - "**.jsx"
+exclude_paths:
+- test/
```

Comment:
@Fleischers I am not sure that we need to make any exceptions for **test** folder, because test code should be also written in the same style and be validated as source code as well.